session.go

```diff
@@ -366,7 +375,6 @@ func (s *Session) mux(remoteBegin *performBegin) {
 					}
 					debug(1, "TX(Session): %s", flow)
 					s.txFrame(flow, nil)
-					remoteOutgoingWindow = s.incomingWindow
```

Comment:
same. we should not update the remoteOutgoingWindow ourselves. trust the protocol :)